python/ray/tests/test_actor_retry_2.py

```diff
@@ -338,5 +339,39 @@ def test_task_retries_on_exit(ray_start_regular_shared):
     ]
 
 
+def test_retry_not_dependent_task_seq_no(ray_start_regular_shared):
```

Comment:
The test name is overfit to the implementation detail. Suggest something like:
```suggestion
def test_retry_dependent_task_on_same_actor(ray_start_regular_shared):
```